### PR TITLE
Yangineer simcity

### LINKED ISSUE #14: What Yoda told yangineer
I heard Yoda saying:
> Too many branches you have
> Delete branch you must

- [x] Merge all branches to master
- [x] Reduce number of branches

### PR DESCRIPTION
**Pull request by yangineer**
Pull this in!

Closes #14

### CODE REVIEW

File: yangineer-simcity.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Famous Landmarks
+- Coal power plant
+- Gas power plant
+- Police station
+- Fire station
+
+## Specialities
+1. Reticulating splines
+1. Disasters:
```
Comment (michellessa2) [inline yangineer-simcity.md:17]: some comment

File: yangineer-simcity.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Simcity
+The famous city from the game Simcity.
+Btw, the last Simcity sucked due to server issues, forced multiplayer mode, and limited city size.
```
Comment (michellessa2) [inline yangineer-simcity.md:3]: I have never played Simcity

**Review by michellessa2 — APPROVED** [review]
great job

File: yangineer-simcity.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+## Restaurants
+- Whatever gets built in the commercial zones
+- Qualitiy depends on how much $$$ the zone is rated
```
Comment (yangineer) [inline yangineer-simcity.md]: I misspelled *Quality*...

File: yangineer-simcity.md (hunk 2 above)
Comment (yangineer) [reply yangineer-simcity.md:3]: It's a city building simulator game made by the same creators of The Sims.
It used to be awesome, until the latest iteration where they forced multiplayer mode on every.

**Pull request merged**